NDTensors/src/lib/TypeParameterAccessors/ext/TypeParameterAccessorFillArraysExt/set_types.jl

```diff
@@ -0,0 +1,10 @@
+using FillArrays: AbstractFill, Fill, Zeros
+using NDTensors.TypeParameterAccessors: Position, default_parameter
+using NDTensors.UnspecifiedTypes: UnspecifiedZero
+
+# ## default parameters
+function default_parameter(::Type{<:AbstractFill}, ::Position{1})
+  return UnspecifiedZero
+end
+default_parameter(::Type{<:AbstractFill}, ::Position{2}) = 0
+default_parameter(::Type{<:AbstractFill}, ::Position{3}) = Tuple{}
```

Comment:
Base on functions (`eltype`, `ndims`, etc.) instead of `Position`.

Reply:
I updated this to use the functions. for `Position{3}` I have to import the `axestype` function from `UnallocatedArrays`